experiments/test/cli.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, test, expect } from "vitest"
+import { runner } from "clet"
+import fsExists from "fs.promises.exists"
+
+describe("cli", () => {
+  test(
+    "descriptor codegen",
+    async () => {
+      await runner().spawn("polkadot-api", ["--key polkadot"], {}).code(0).end()
```

Comment:
To not do `pnpm link --global` in the GH workflow, Why not spawn `pnpm polkadot-api`?

`@polkadot-api/cli` is already a dependency in experiments, so it may work